src/bin/cargo-flamegraph.rs

```diff
@@ -436,8 +448,14 @@ fn main() -> anyhow::Result<()> {
         opt.package = Some(target.package);
         target.kind
     } else if let Some(unit_test) = opt.unit_test {
+        let kinds: &[&str] = match opt.unit_test_kind {
```

Comment:
Nit: I don't think we need the type annotation here? Would prefer to do without it.

Reply:
I always prefer to avoid those too, but when I first tried without it, I got the following error:
```
error[E0308]: `match` arms have incompatible types
   --> src/bin/cargo-flamegraph.rs:454:21
    |
451 |           let kinds = match opt.unit_test_kind {
    |  _____________________-
452 | |             Some(UnitTestTargetKind::Bin) => &["bin"],
    | |                                              -------- this is found to be of type `&[&str; 1]`
453 | |             Some(UnitTestTargetKind::Lib) => &["lib"],
    | |                                              -------- this is found to be of type `&[&str; 1]`
454 | |             None => &["bin", "lib"],
    | |                     ^^^^^^^^^^^^^^^ expected an array with a fixed size of 1 element, found one with 2 elements
455 | |         };
    | |_________- `match` arms have incompatible types
```

Reply:
Yes! Should I push that change? Also, should I add a comment about this on that line? Seems a little "cryptic" to me, I probably wouldn't understand why the `[..]` is there just by looking at the code :sweat_smile:

Reply:
Comment is fine if you think it helps. Please squash this change into the existing commit.

Reply:
Done!